examples/LossMinimization/loss_minimization.jl

```diff
@@ -1,122 +1,201 @@
-using Distributions
-using LinearAlgebra
-using Random
-using Plots
-using EnsembleKalmanProcesses.EnsembleKalmanProcessModule
-using EnsembleKalmanProcesses.ParameterDistributionStorage
-
-# Seed for pseudo-random number generator for reproducibility
+# # Minimization Loss
+#
+# First we load the required packages.
+
+using
+    Distributions,
+    LinearAlgebra,
+    Random,
+    Plots
+
+using
+    EnsembleKalmanProcesses.EnsembleKalmanProcessModule,
+    EnsembleKalmanProcesses.ParameterDistributionStorage
+
+# ## Loss function with single minimum
```

Comment:
```suggestion
# ## Loss function with single minimum
#
# Here we minimize the loss function
# ```math
# G₁(u) = \|u - u_*\|,
# ```
# 
#  where ``u`` is a vector of parameters and ``u_*`` is a vector of optimal parameters
```

As a side note, it might make sense to use a different symbol than `u` since this package is intended for use with fluid dynamical problems...